mantis-client/src/main/java/io/mantisrx/client/MantisSSEJob.java

```diff
@@ -91,13 +90,7 @@ public String getJobId() {
 
     @Deprecated
     public synchronized Observable<MantisServerSentEvent> connectAndGetObservable() throws IllegalStateException {
-        return connectAndGet()
-                .flatMap(new Func1<Observable<MantisServerSentEvent>, Observable<MantisServerSentEvent>>() {
-                    @Override
-                    public Observable<MantisServerSentEvent> call(Observable<MantisServerSentEvent> o) {
-                        return o;
-                    }
-                });
+        return connectAndGet().flatMap(o -> o);
```

Comment:
In general it makes perfect sense. The Java built-in `Function::identity` is very stable across JDK versions since it's introduced. It is well defined and we can use it when it applies.

However, this `connectAndGet()` method returns an Observable object, which has its own `Func1` interface as the `flatMap(Func1)` parameter type. Sadly it's not interoperable with Java built-in `Function` interface. So we can not use it here :(

I did not find reactive Function(s)::identity method; and it is not necessarily better than `x ->x` even they provide one.